examples/subscribing-example-app/public/RiderConnection.js

```diff
@@ -52,6 +55,15 @@ export class RiderConnection {
     this.subscriber.start(channelId || 'ivan');
   }
 
+  setRenderSkippedLocations(state) {
+    this.renderSkippedLocations = state;
+  }
+
+  setSkippedLocationInterval(interval) {
+    console.log(`setting skipped location interval to ${interval}`);
```

Comment:
In particular for browser runtime, I'm always a bit cautious of any kind of console logging - especially when it's using raw APIs which can't be effectively disabled at runtime without removing code.

I notice we have [a `Logger` class defined](https://github.com/ably/ably-asset-tracking-js/blob/c35753121c46464cd0b3b694d0ec5b1d09b630da/src/lib/utils/Logger.ts#L13) in this codebase. Is there a reason you cannot use that here?

On [a quick search](https://github.com/ably/ably-asset-tracking-js/search?q=console.log), it doesn't seem that we use `console.log` elsewhere right now in this app.

Reply:
My bad, I was just using this for local debugging, definitely didn't mean to push it up. d200f5d.